drivers/power/ltc3350/ltc3350.h

```diff
@@ -0,0 +1,235 @@
+#define LTC3350_AD_DTEMP_COLD_LVL               0x13
+#define LTC3350_AD_DTEMP_HOT_LVL                0x14
+#define LTC3350_AD_ESR_HI_LVL                   0x15
+#define LTC3350_AD_CAP_LO_LVL                   0x16
+#define LTC3350_AD_CTL_REG                      0x17
+#define LTC3350_AD_NUM_CAPS                     0x1A
+#define LTC3350_AD_CHRG_STATUS                  0x1B
+#define LTC3350_AD_MON_STATUS                   0x1C
+#define LTC3350_AD_ALARM_REG                    0x1D
+#define LTC3350_AD_MEAS_CAP                     0x1E
+#define LTC3350_AD_MEAS_ESR                     0x1F
+#define LTC3350_AD_MEAS_VCAP1                   0x20
+#define LTC3350_AD_MEAS_VCAP2                   0x21
+#define LTC3350_AD_MEAS_VCAP3                   0x22
+#define LTC3350_AD_MEAS_VCAP4                   0x23
+#define LTC3350_AD_MEAS_GPI                     0x24
+#define LTC3350_AD_MEAS_VIN                     0x25
+#define LTC3350_AD_MEAS_VCAP                    0x26
+#define LTC3350_AD_MEAS_VOUT                    0x27
+#define LTC3350_AD_MEAS_IIN                     0x28
+#define LTC3350_AD_MEAS_ICHG                    0x29
+#define LTC3350_AD_MEAS_DTEMP                   0x2A
+
+/******************************************************************************/
+/******************* Driver pre-compile time settings *************************/
+/******************************************************************************/
+
+/**
+ * @brief   LTC3350 measurements configuration switch.
+ * @details If set to @p TRUE more configurations are available.
+ * @note    The default is @p TRUE.
+ */
+#ifndef LTC3350_USE_MEASUREMENTS
+#define LTC3350_USE_MEASUREMENTS               	1
+#endif
+
+/**
+ * @brief   LTC3350 alarms configuration switch.
+ * @details If set to @p TRUE more configurations are available.
+ * @note    The default is @p FALSE.
+ */
+#ifndef LTC3350_USE_ALARMS
+#define LTC3350_USE_ALARMS               		0
+#endif
+
+/******************************************************************************/
+/*************************** Types Declarations *******************************/
+/******************************************************************************/
+
+/*LTC3350 alarms*/
+enum ltc3350_enum_alarms {
+	LTC3350_CAP_UV_LVL            = 0,
+	LTC3350_CAP_OV_LVL            = 1,
+	LTC3350_GPI_UV_LVL            = 2,
+	LTC3350_GPI_OV_LVL            = 3,
+	LTC3350_VIN_UV_LVL            = 4,
+	LTC3350_VIN_OV_LVL            = 5,
+	LTC3350_VCAP_UV_LVL           = 6,
+	LTC3350_VCAP_OV_LVL           = 7,
+	LTC3350_VOUT_UV_LVL           = 8,
+	LTC3350_VOUT_OV_LVL           = 9,
+	LTC3350_IIN_OC_LVL            = 10,
+	LTC3350_ICHG_UC_LVL           = 11,
+	LTC3350_DTEMP_COLD_LVL        = 12,
+	LTC3350_DTEMP_HOT_LVL         = 13,
+	LTC3350_ESR_HI_LVL            = 14,
+	LTC3350_CAP_LO_LVL            = 15
+};
+
+/*LTC3350 alarms_mask*/
+enum ltc3350_enum_alarms_mask {
+	LTC3350_CAP_UV_LVL_B_BIT      = NO_OS_BIT(0),
```

Comment:
this enum can be dropped completely and use `NO_OS_BIT` inline on the `enum ltc3350_enum_alarms`. You already do something like this here:
https://github.com/analogdevicesinc/no-OS/pull/1920/files#diff-4979d48b026f39fd88186df28e27f214871d4b933cbd03664bd2842c4609067cR274